.travis.yml

```diff
@@ -26,6 +26,7 @@ env:
   - HELLBENDER_TEST_INPUTS=gs://hellbender/test/resources/
   - HELLBENDER_TEST_STAGING=gs://hellbender/test/staging/
   - HELLBENDER_TEST_PROJECT=broad-dsde-dev
+  - HELLBENDER_JSON_SERVICE_ACCOUNT_KEY=servicekey.json
```

Comment:
Is `GOOGLE_APPLICATION_CREDENTIALS` how the "default" credentials are passed in? We thought that maybe you were trying to separate passing of "default" credentials from passing of explicit credentials -- is that a concern here?